shared/src/business/entities/trialSessions/TrialSession.js

```diff
@@ -14,9 +16,29 @@ const {
   US_STATES_OTHER,
 } = require('../EntityConstants');
 const { createISODateString } = require('../../utilities/DateHandler');
-const { isEmpty } = require('lodash');
+const { isEmpty, isEqual } = require('lodash');
 const { JoiValidationConstants } = require('../JoiValidationConstants');
 
+const getTrialSessionStatus = session => {
+  const allCases = session.caseOrder || [];
+  const inactiveCases = allCases.filter(
+    sessionCase => sessionCase.removedFromTrial === true,
+  );
+
+  if (
+    session.isClosed ||
```

Comment:
This should be removed now, right?

Reply:
we use that helper function for the migration script, so we can't remove that function yet

Reply:
Ah, ok. If it's only needed for the migration, I would prefer that logic move into the migration script itself so it gets cleaned up when the migration script is deleted. Just my opinion, I won't hold up the PR for that though.

Reply:
actually I was mistaken, we still use that logic in the removeCaseFromCalendar method.  When recompute and store the status when you remove a case from a trial session.